R/members.R

```diff
@@ -256,14 +256,18 @@ space_member_remove.data.frame <- function(space, users, remove_projects = NULL,
 #'
 #' Returns the list of members and their usage information for a given space.
 #' You must either be the admin of the space or your role must have permissions
-#' to see the members list.
+#' to see the members list. Result will include `last_activity` only if only
+#' if the time window is less than or equal to 31 days.
 #'
 #' @inheritParams space_info
 #' @inheritParams rscloud_space_list
 #'
 #' @examples
 #' \dontrun{
-#' # Usage in the last 90 days for each user
+#' # Usage in the last 30 days for each user, will report last_activity
+#' space_member_usage(space, filters = list(groupby = "user_id", from = "30d"))
+#'
+#' # Usage in the last 90 days for each user, will not report last_activity
```

Comment:
Is `last_activity` defined anywhere? It's not clear to me, but it may be clear to users of the package 🤷‍♂️  Might be good to make clear that it is the `last_activity` "column"

Reply:
Described what `last_activity` means